cmd/crio/config.go

```diff
@@ -131,6 +131,13 @@ image_volumes = "{{ .ImageVolumes }}"
 insecure_registries = [
 {{ range $opt := .InsecureRegistries }}{{ printf "\t%q,\n" $opt }}{{ end }}]
 
+# registries is used to specify registries to be used when pulling an unqualified
+# image (e.g. fedora:rawhide). "docker.io" is left here as a default registry.
+# Users can remove it and use whatever default registry they need.
+registries = [
+  "docker.io",
```

Comment:
@rhatdan the double "docker.io" issue is not an issue (the last command totally overrides the docker.io entry also):

```
$ cat test.config| grep docker.io
  "docker.io",

$ sudo ./crio --config test.config --debug --registry docker.io --registry test.redhat.com
DEBU[2017-08-01 10:28:44.630508363+02:00] runcom [docker.io test.redhat.com]

$ sudo ./crio --configtest.config --debug --registry test.rhel.com
DEBU[2017-08-01 10:29:49.679527671+02:00] runcom [test.rhel.com]
```
You can see docker.io isn't added twice.